docs/content/direct/multi-wec-aggregated-status.md

```diff
@@ -0,0 +1,191 @@
+# Proposal: Multi-WEC Aggregated Status Controller Enhancement
+
+**Status:** Proposal  
+**Related Issue:** [#2809 – KubeStellar controller logic to map back the status from WECs to READY values in WDSes](https://github.com/kubestellar/kubestellar/issues/2809)
+
+---
+
+## Current Scenario
+
+In the current KubeStellar implementation:
+
+### Case 1 — Single WEC and `wantSingletonReportedState` is true  
+The WDS workload `.status` is propagated from the single WEC.
+
+### Case 2 — Multiple WECs and `wantSingletonReportedState` is false  
+For workloads deployed to multiple WECs, the `.status` field in the WDS remains empty, resulting in no aggregated status propagated.
+
+This leads to limited visibility in ArgoCD, where workloads appear as *OutOfSync* or *Unknown*, even though all WECs are healthy.
+
+**Reference:**  
+- [PR #3297](https://github.com/kubestellar/kubestellar/pull/3297)  
+- [Multi-cluster example in docs](https://docs.kubestellar.io/release-0.28.0/direct/example-scenarios/#scenario-1-multi-cluster-workload-deployment-with-kubectl)
+
+---
+
+## Goal
```

Comment:
This section needs to say that the requirement is to be good enough to make ArgoCD happy, and there is a stretch goal to do something somewhat useful for additional positions.